deploy/examples/cluster.yaml

```diff
@@ -88,6 +88,10 @@ spec:
       # Requires Ceph Quincy (v17) or newer. Also see the kernel requirements above for encryption.
       compression:
         enabled: false
+      # Whether to require communication over msgr v2. If true, the msgr v1 port will be disabled
```

Comment:
Not possible to configure encryption support in helm values.yaml?

Reply:
Yes, anything in the cluster spec can be added in the `cephClusterSpec` of the cluster chart. I'll add it there just to be clear, thanks.